montecarlo/node.py

```diff
@@ -70,7 +70,7 @@ def get_score(self, root_node):
         return self.score
 
     def is_scorable(self):
-        return (self.visits or self.policy_value != None) and not self.is_widen_node
+        return self.visits or self.policy_value != None or self.is_widen_node
```

Comment:
can we have a comment, i am getting confused

Reply:
Good idea, added